ads/google/a4a/traffic-experiments.js

```diff
@@ -165,9 +173,24 @@ function maybeSetExperimentFromUrl(win, element, experimentName,
     '0': null, // TODO Ensure does not generate exp id
     '1': controlBranchId,
     '2': treatmentBranchId,
+    '3': hasLaunched(win, element) ? treatmentBranchId : controlBranchId, // DF
```

Comment:
I'm not 100% sure this (always DF regardless of launch affecting whether that's control or treatment) is the desired approach. Please confirm?